localization/yabloc/yabloc_particle_filter/src/gnss_corrector/gnss_corrector_core.cpp

```diff
@@ -96,10 +98,10 @@ void GnssParticleCorrector::process(
   // Compute travel distance from last update position
   // If the distance is too short, skip weighting
   {
-    Eigen::Vector3f mean_position = common::pose_to_affine(mean_pose).translation();
-    if ((mean_position - last_mean_position_).squaredNorm() > 1) {
+    Eigen::Vector3f tmp_mean_position = common::pose_to_affine(mean_pose).translation();
+    if ((tmp_mean_position - last_mean_position_).squaredNorm() > 1) {
       this->set_weighted_particle_array(weighted_particles);
-      last_mean_position_ = mean_position;
+      last_mean_position_ = tmp_mean_position;
```

Comment:
@KYabuuchi this is also avoiding variable shadowing, but it needs the value of `tmp_mean_position`  to be stored for efficiency.

Reply:
@a-maumau Since `mean_pose` in line 88 and `tmp_mean_pose` are the same value, we don't need to declare it anyway. Sorry for the confusing implementation. :bow:

Reply:
I fixed it at 31e2994 .